app/src/main/java/com/codesoom/assignment/http/HttpResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.http;
+
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.IOException;
+
+public abstract class HttpResponse implements HttpResponseSend{
+
+	private HttpExchange httpExchange;
+	private int statusCode;
+
+	protected HttpResponse(HttpExchange httpExchange, int statusCode) {
+		this.httpExchange = httpExchange;
+		this.statusCode = statusCode;
+	}
+
+	public void send() throws IOException {
+		sendResponse();
+	}
+
+	public void send(String responseBody) throws IOException {
+		sendResponse(responseBody);
+	}
+
+	private void sendResponse(String responseBody) throws IOException {
+		httpExchange.sendResponseHeaders(statusCode, responseBody.getBytes().length);
+		httpExchange.getResponseBody().write(responseBody.getBytes());
+		httpExchange.getResponseBody().close();
+	}
+
+	private void sendResponse() throws IOException {
+		httpExchange.sendResponseHeaders(statusCode, 0);
+		httpExchange.close();
+	}
```

Comment:
sendResponse라는 프라이빗 메서드를 만드셨네요. 이 프라이빗 메서드는 없어도 괜찮을 것 같네요. 뭔가 다른 로직을 재사용한다던지, 아니면 추상화가 더 필요하다던지 다른 이유가 있으면 더 좋을 것 같아요

```suggestion
	public void send() throws IOException {
		httpExchange.sendResponseHeaders(statusCode, 0);
		httpExchange.close();
	}

	public void send(String responseBody) throws IOException {
		httpExchange.sendResponseHeaders(statusCode, responseBody.getBytes().length);
		httpExchange.getResponseBody().write(responseBody.getBytes());
		httpExchange.getResponseBody().close();
	}
```

Reply:
앗 불필요하게 작성되었네요^^; 제거했어요. 감사합니다